roadmap/implementors-guide/guide.md

```diff
@@ -989,12 +991,78 @@ Dispatch a `PovFetchSubsystemMessage(relay_parent, candidate_hash, sender)` and
 
 (TODO: send statements to Statement Distribution subsystem, handle shutdown signal from candidate backing subsystem)
 
+### Statement Gossip Subsystem
+
+The statement gossip subsystem is responsible for gossiping out and receiving [Signed Statements](#Signed-statement-type) from validators.
+
+#### Message Types
+
+This subsystem communicates with the overseer via [two message types](#Candidate-Backing-Subsystem-Messages), one for overseer -> subsystem and another for subsystem -> overseer. Currently this just differentiates between statements that the subsystem has been instructed to gossip out, and statements that the subsystem has received.
+
+#### Functionality
+
+The subsystem needs to contain a handle to a gossiping engine to gossip and recieve messages. This will be cloned into each job that is spawned. Apart from that, it also contains the general structures that all subsystems contain, e.g. channels to communicate with the overseer and handles to spawned jobs in order to shut them down.
+
+On `OverseerSignal::StartWork` it should:
+* Spawn a job and pass in `relay_parent`, a clone of the gossiping engine and a handle to a message validator.
+* Send out a neighbour packet with the `relay_parent` added to the list of accepted chain heads.
```

Comment:
In case you agree with me that the neighbor packet should be sent via the `GossipEngine` and not via the `Validator`, would you mind adding a note here?

If there is no agreement I can expand on why this would help the architecture.

Reply:
One thing we need to be aware of is that the validator needs to be aware of the most recent neighbor packets we have sent to peers. I suppose we just need to be careful when the `GossipEngine` is polled when doing that update.